app/models/tutorial.rb

```diff
@@ -3,7 +3,9 @@ class Tutorial < ApplicationRecord
 
   friendly_id :title
 
-  has_many :tutorial_steps, dependent: :destroy
+  has_many :tutorial_steps, -> { order(:id) }, dependent: :destroy, inverse_of: :tutorial
```

Comment:
yes but IDs are assigned chronologically. this is what tutorials was previously implicitly relying on, except that it didn't even order by ID so sometimes it was out of order (because of SQL)

Reply:
I'm uncomfortable relying on the assumption that IDs will be in the order we want. For now, sure, that's always true, but someday there will be a CMS of some kind, and someday a different team will be working on this site, and they may not realize that Tutorials relies on this behavior.

Would the correct approach be to add some sort of index or sequence column? Maybe this means adding an issue so that this PR doesn't get stopped in its tracks.